### PR TITLE
[Decode] Check whether the vui_ext_num_entries exceeds the specified value

### PR DESCRIPTION
**Pull request by chuanli1**
Issue: MDP-68766
Test: manual

### CODE REVIEW

File: _studio/shared/umc/codec/h264_dec/src/umc_h264_dec_bitstream_headers.cpp [hunk 1]
```diff
@@ -1006,7 +1006,8 @@ Status H264HeadersBitstream::GetSequenceParamSetSvcVuiExt(H264SeqParamSetSVCExte
 {
     SVC_VUI vui;
     vui.vui_ext_num_entries = GetVLCElement(false) + 1;
-
+    if(vui.vui_ext_num_entries > 1024)
```
Comment (wangyan-intel) [inline _studio/shared/umc/codec/h264_dec/src/umc_h264_dec_bitstream_headers.cpp:1009]: Could we use macro instead of 1024?

**Review by wangyan-intel — CHANGES_REQUESTED** [review]

**Review by wangyan-intel — APPROVED** [review]

**Comment by chuanli1** [comment]
Hi Dmitry, could you merge this PR? Thanks.

**Review by dmitryermilov — APPROVED** [review]
LGTM

**Comment by dmitryermilov** [comment]
> @dmitryermilov, Hi Dmitry, could you review this PR? Thanks.

Done!

**Pull request merged**